pkg/apis/crd/v1alpha1/types.go

```diff
@@ -434,6 +434,11 @@ type NetworkPolicyPeer struct {
 	//  Exact FQDNs, i.e. "google.com", "db-svc.default.svc.cluster.local"
 	//  Wildcard expressions, i.e. "*wayfair.com".
 	FQDN string `json:"fqdn,omitempty"`
+	// Select all Pods with the ServiceAccount matched by this field, as
+	// workloads in AppliedTo/To/From fields.
+	// Cannot be set with any other selector.
+	// +optional
+	ServiceAccounts []ServiceAccount `json:"serviceAccounts,omitempty"`
```

Comment:
Will we be adding this to `AppliedTo` as well? If not, any reason why not?

Reply:
Thanks for pointing it out. Yes, we should add it to `AppliedTo` as well.

Reply:
The `AppliedTo `struct is used by `Egress ` only today (though I do hope we change NetworkPolicy and Rule to use `AppliedTo` for the AppliedTo field.